src/fragments/lib-v1/datastore/flutter/relational/save-many-snippet.mdx

```diff
@@ -1,25 +1,36 @@
 Add the model above to the `schema.graphql` file located by default at `amplify/backend/{api_name}/` and regenerate the models again with the following command:
 
-```console
+```bash
```

Comment:
We seem to be split 50/50 across the docs on using bash/console. Unless there is a benefit to using bash, I suggest reverting this change to limit the scope of changes in this large PR.